tests/app/dao/test_ft_billing_dao.py

```diff
@@ -1245,22 +1248,22 @@ def test_fetch_volumes_by_service(notify_db_session):
     assert results[1].letter_sheet_totals == 0
     assert float(results[1].letter_cost) == 0
 
-    assert results[2].service_name == 'b - chargeable sms'
-    assert not results[2].organisation_name
-    assert results[2].free_allowance == 10
-    assert results[2].sms_notifications == 2
-    assert results[2].sms_chargeable_units == 3
-    assert results[2].email_totals == 0
-    assert results[2].letter_totals == 0
-    assert results[2].letter_sheet_totals == 0
-    assert float(results[2].letter_cost) == 0
-
-    assert results[3].service_name == 'e - sms within allowance'
-    assert not results[3].organisation_name
-    assert results[3].free_allowance == 10
-    assert results[3].sms_notifications == 1
-    assert results[3].sms_chargeable_units == 2
-    assert results[3].email_totals == 0
-    assert results[3].letter_totals == 0
-    assert results[3].letter_sheet_totals == 0
-    assert float(results[3].letter_cost) == 0
+    assert results[4].service_name == 'b - chargeable sms'
```

Comment:
Why aren't we checking the contents of `results[2]`, `results[3]` and `results[5]` here? I'd like a code comment to explain why please I think

Reply:
decision: leave a comment in test explaining why we're not checking all rows

Reply:
Done ^_^